src/templates/base/2019/base_ebook.html

```diff
@@ -27,125 +32,127 @@
 {% endblock %}
 
 {% block content %}
-<section class="content">
+<div class="content">
```

Comment:
I'd love to see the cover page stand out more (maybe darker background color, white/yellow text) and give "2019" more prominence, using the home page design as inspiration:

![image](https://user-images.githubusercontent.com/1120896/82123886-d3438800-9769-11ea-9388-65751a026133.png)
![image](https://user-images.githubusercontent.com/1120896/82123893-d9396900-9769-11ea-8982-eb91395b0960.png)

Reply:
Uploaded a new version there. Let me know your thoughts.

Not done the diagonal stripe yet though, but gives some idea of what it might look like.

Reply:
Diagonal stripe added by re-using our SVG.